astropy/io/ascii/core.py

```diff
@@ -259,19 +271,13 @@ class DefaultSplitter(BaseSplitter):
           for col_val in col_vals:
                ...
 
-    :param delimiter: one-character string used to separate fields.
-    :param doublequote:  control how instances of *quotechar* in a field are quoted
-    :param escapechar: character to remove special meaning from following character
-    :param quotechar: one-character stringto quote fields containing special characters
-    :param quoting: control when quotes are recognised by the reader
-    :param skipinitialspace: ignore whitespace immediately following the delimiter
     """
-    delimiter = ' '
-    quotechar = '"'
-    doublequote = True
-    escapechar = None
-    quoting = csv.QUOTE_MINIMAL
-    skipinitialspace = True
+    delimiter = ' '  #: one-character string used to separate fields.
+    quotechar = '"'  #: control how instances of *quotechar* in a field are quoted
+    doublequote = True  #: character to remove special meaning from following character
+    escapechar = None  #: one-character stringto quote fields containing special characters
+    quoting = csv.QUOTE_MINIMAL  #: control when quotes are recognised by the reader
+    skipinitialspace = True  #: ignore whitespace immediately following the delimiter
```

Comment:
I have no preference.  The other syntax was just the first one shown in the sphinx doc example.  If the triple-quoted string below works then that's just dandy.